gluetool/glue.py

```diff
@@ -878,6 +950,7 @@ class Glue(Configurable):
     """
 
     name = 'gluetool core'
+    unique_name = 'gluetool core'
```

Comment:
Sorry but I haven't caught what is this variable used for, could you provide my some help?

Reply:
`unique_name` is the name under which the module is known in the pipeline, one of its many possible aliases. Each instance of module can have different `unique_name` but they share the same `name`.

Reply:
ok [RESOLVED]